components/server/src/routes/auth.py

```diff
@@ -134,3 +135,11 @@ def logout(database: Database) -> dict[str, bool]:
     """Log the user out."""
     delete_session(database)
     return dict(ok=True)
+
+
+@bottle.get("/api/v3/public_key")
+def get_public_key(database: Database) -> dict:
+    """Return a serialized version of the public key."""
+    public_key = database.secrets.find_one({"name": EXPORT_FIELDS_KEYS_NAME}, {"public_key": True, "_id": False})
```

Comment:
Shouldn't we add `"private_key": False` so the private key isn't needlessly passed to the server?

Reply:
That's not necessary. From the mongo docs I read that if a field is explicitly included, al non-mentioned field will be automatically excluded. With the exception of _id which needs to be excluded explicitly.

https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/

Reply:
Ah, didn't know that. Good to know.